.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+	"solidity.compileUsingRemoteVersion": "v0.6.11+commit.5ef660b1"
```

Comment:
current version  is 0.8.18. Why not the latest version?

Reply:
Snarkjs version 0.5.0 (stable version) automatically generates the solidity contract and uses the 0.6.11 compiler version. The latest snarkjs version (0.7.0) uses compiler version 0.8.18 but the code is unstable.